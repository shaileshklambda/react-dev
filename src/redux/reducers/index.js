/*
    Uses combineReducers Utility To Combine all The Reducers.
*/

import { combineReducers } from "redux";
import postReducer from "./postReducer";
import apiReducer from "./apiReducer";
import timeline from "./timeline";
import debuggerReducer from "./debuggerReducer";
export default combineReducers({
  apiReducer,
  postReducer,
  timeline,
  debuggerReducer
});
