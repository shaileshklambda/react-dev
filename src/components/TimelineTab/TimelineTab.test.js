import React from "react";
import renderer from "react-test-renderer";
import Timelinetab from ".";
import { store } from "../../redux/store";

it("Timelinetab: default", () => {
  const component = renderer.create(<Timelinetab store={store} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
