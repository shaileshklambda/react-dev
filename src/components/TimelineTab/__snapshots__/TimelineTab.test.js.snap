// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timelinetab: default 1`] = `
<div
  className="text-center"
>
  No Data to render
</div>
`;
